common/djangoapps/third_party_auth/api/tests/test_views.py

```diff
@@ -247,26 +242,6 @@ def test_list_all_user_mappings_withapi_key(self, api_key, provider_id, expect_c
         response = self.client.get(url, HTTP_X_EDX_API_KEY=api_key)
         self._verify_response(response, expect_code, expect_data)
 
-    @ddt.data(
-        (PROVIDER_ID_TESTSHIB, 'valid-token', 200, get_mapping_data_by_usernames(LINKED_USERS)),
-        ('non-existing-id', 'valid-token', 404, []),
-        (PROVIDER_ID_TESTSHIB, 'invalid-token', 401, []),
-    )
-    @ddt.unpack
-    def test_list_all_user_mappings_oauth2(self, provider_id, access_token, expect_code, expect_data):
```

Comment:
Why is this test being removed?

Reply:
I originally thought it was not necessary cause the new permission class only works with JWT token. But now i realize I should recreate test using JWT auth instead. The problem is that I don't know how to create jwt token correctly with the required scopes and filters. please help

Reply:
nvm, I think I found solution using from generate_jwt from edx_rest_framework_extensions.auth.jwt.tests.utils

Reply:
figured it out